style.css

```diff
@@ -0,0 +1,1226 @@
+  .search-menu li {
+    border-left: 1px solid #eee;
+    padding-left: 10px;
+  }
+
+  .article-details ul li {
+    margin-left: 0;
+    margin-right: 5px;
+  }
+
+  .search-menu li:nth-child(2) {
+    display: none;
+  }
+
+  #main-menu-links li:nth-last-child(2),
+  #main-menu-links li:nth-last-child(1) {
+    display: none;
+  }
+
+  a {
+    font-family: sans-serif;
+    font-weight: 500;
+    font-size: 13px;
+    color: #aaa;
+  }
+
+  nav {
+    width: 100%;
+    height: 5rem;
+    display: flex;
+    color: #eee;
+    z-index: 999;
+  }
+
+  .tnw-logo img {
+    height: 1rem;
+    width: 66px;
+    padding: 20px;
+  }
+
+  .tnw-logo {
+    height: 100%;
+    width: 106px;
+  }
+
+  .menu-container {
+    width: 100%;
+  }
+
+  .top-menu {
+    display: flex;
+    height: 2rem;
+  }
+
+  .main-menu {
+    position: sticky;
+    top: 0;
+    display: flex;
+    width: 100%;
+    height: 3rem;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    color: black;
+    z-index: 999;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 0;
+  }
+
+  .main-menu ul {
+    display: flex;
+  }
+
+  .search-menu {
+    padding-right: 20px;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .secondary-menu {
+    position: relative;
+    display: flex;
+    font-size: 13px;
+    font-weight: 500;
+    width: 100%;
+    height: 2rem;
+    background: #131313;
+    align-items: center;
+    padding: 0 10px;
+    box-sizing: border-box;
+  }
+
+  .secondary-1 {
+    min-width: 427px;
+  }
+
+  .secondary-2 {
+    float: right;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .secondary-menu div {
+    position: absolute;
+    right: 12px;
+  }
+
+  .cover-story-article ul {
+    display: flex;
+  }
+
+  .secondary-menu div ul {
+    display: flex;
+  }
+
+  .cover-story {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    grid-gap: 4px;
+    height: 45vh;
+    padding: 4px;
+  }
+
+  .cover-story-article {
+    min-height: auto;
+    position: relative;
+    padding: 4px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .cover-story-article header {
+    color: white;
+    padding: 14px 12px 12px 12px;
+    z-index: 2;
+  }
+
+  .cover-story-article h2 {
+    font-size: 20px;
+    font-weight: 700;
+  }
+
+  .cover-story-article h4 {
+    font-size: 14px;
+  }
+
+  .story1 {
+    grid-column: 1/span 2;
+    grid-row: 1/span 2;
+  }
+
+  .story1 h2 {
+    font-size: 48px;
+    font-weight: 700;
+  }
+
+  .story-background {
+    background-size: 100% 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: -20;
+  }
+
+  .gradient-background {
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 1;
+    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 1));
+  }
```

Comment:
- [ ] The same two classes at the beginning. You could do the same in this media query and put them together and then specify the particular details.

Reply:
Thanks, I merged them together already